docs/draft_production_docs/oadp-self-service.md

```diff
@@ -0,0 +1,511 @@
+    bucket: my-company-bucket <---
+    prefix: velero
+  provider: aws
+```
+The DPA spec must be set in the following way: 
+
+```
+nonAdmin:
+  enable: true
+  enforceBSLSpec:
+    config:             <--- entire config must match expected NaBSL config
+      checksumAlgorithm: ""
+      profile: default
+      region: us-west-2
+    objectStorage:  <--- all of the objectStorage options must match expected NaBSL options
+      bucket: my-company-bucket
+      prefix: velero 
+    provider: aws
+```
+
+#### Restricted NonAdminBackups
+
+In the same sense as the NABSL, cluster administrators can also restrict the NonAdminBackup spec fields to ensure the backup request conforms to the administrator defined policy.  Most of the backup spec fields can be restricted by the cluster administrator, below is a table of reference for the current implementation.
+
+
+| **Backup Spec Field**                                  | **Admin Enforceable** | **Restricted** | **special case** |
+|--------------------------------------------|--------------|--------------------------|-----------------|
+| `csiSnapshotTimeout`                       | ✅ Yes       |                          |                 |
+| `itemOperationTimeout`                     | ✅ Yes       |                          |                 |
+| `resourcePolicy`                           | ✅ Yes       |                          | ⚠️ Non-admin users can specify the config-map that admins created in OADP Operator NS(Admins enforcing this value be a good alternative here), they cannot specify their own configmap as its lifecycle handling is not currently managed by NAC controller |
+| `includedNamespaces`                       | ❌ No        | ✅ Yes                   | ⚠️ Admins cannot enforce this because it does not make sense for a cluster wide non-admin backup setting, we have validations in place such that only the NS admins NS in included in the NAB spec.                 |
+| `excludedNamespaces`                       | ✅ Yes       | ✅ Yes                   | ⚠️ This spec is restricted for non-admin users and hence not enforceable by admins                |
+| `includedResources`                        | ✅ Yes       |                          |                 |
+| `excludedResources`                        | ✅ Yes       |                          |                 |
+| `orderedResources`                         | ✅ Yes       |                          |                 |
+| `includeClusterResources`                  | ✅ Yes       |                          | ⚠️ Non-admin users can only set this spec to false if they want, all other values are restricted, similar rule for admin enforcement regarding this spec value.  |
+| `excludedClusterScopedResources`           | ✅ Yes       |                          |                 |
+| `includedClusterScopedResources`           | ✅ Yes       |                          | ⚠️ This spec is restricted and non-enforceable, only empty list is acceptable  |
```

Comment:
Should have Restricted column Yes? https://github.com/migtools/oadp-non-admin/blob/af1bbcd371bfd7c7117b0aca93105e16c7ddefb1/internal/common/function/function.go#L122